frontend/app/.server/domain/mappers/benefit-renewal.dto.mapper.ts

```diff
@@ -180,13 +179,13 @@ export class DefaultBenefitRenewalDtoMapper implements BenefitRenewalDtoMapper {
       return {};
     }
 
-    const { hasAddressChanged, hasEmailChanged, hasFederalBenefitsChanged, hasMaritalStatusChanged, hasPhoneChanged, hasProvincialTerritorialBenefitsChanged } = changeIndicators;
+    const { hasAddressChanged, hasEmailChanged, hasMaritalStatusChanged, hasPhoneChanged, hasFederalProvincialTerritorialBenefitsChanged } = changeIndicators;
     return {
       AddressChangedIndicator: hasAddressChanged,
       EmailChangedIndicator: hasEmailChanged,
       MaritalStatusChangedIndicator: hasMaritalStatusChanged,
       PhoneChangedIndicator: hasPhoneChanged,
-      PublicInsuranceChangedIndicator: !!hasFederalBenefitsChanged || !!hasProvincialTerritorialBenefitsChanged,
+      PublicInsuranceChangedIndicator: !!hasFederalProvincialTerritorialBenefitsChanged,
```

Comment:
Nitpick: Since `hasFederalProvincialTerritorialBenefitsChanged` is already a `boolean`

```suggestion
      PublicInsuranceChangedIndicator: hasFederalProvincialTerritorialBenefitsChanged,
```


FYI the use of `!!` was needed in the original code because both `hasFederalBenefitsChanged` and `hasProvincialTerritorialBenefitsChanged` needed to be explicitly `boolean` before using `||`